kits/democracy/scripts/deploy.js

```diff
@@ -22,5 +27,9 @@ module.exports = async (callback) => {
   // aragonID
   await deploy_id(null, { artifacts, web3, ensAddress: ens.address })
 
-  await deploy_kit(null, { artifacts, kitName: 'democracy-kit', kitContractName: 'DemocracyKit', network: network, ensAddress: ens.address })
+  await deploy_kit(null, { artifacts, kitName: 'democracy-akropolis', kitContractName: 'DemocracyKit', network: network, ensAddress: ens.address })
```

Comment:
Just curious, why this change? Anyway, it's definitely nicer! ;-) (although maybe less intuitive, and I see mltisig one hasn't been changed)

Reply:
My bad! 🤦🏼‍♂️
I carried a change from another fork, please merge #81 to fix it 🙏🏻